extensions/blocks/revue/view.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { each } from 'lodash';
```

Comment:
I wonder if there would be a way to do without `each` here, to try to reduce the size of the js we'll need to enqueue on all pages with a form?

Reply:
Fixed in b49c0bf68dcf4689410448e47390fb48bd43badf